composer.json

```diff
@@ -21,8 +22,17 @@
       "OpenEuropa\\CodeReview\\Tests\\": "./tests/"
     }
   },
+  "extra": {
+    "enable-patching": true,
+    "composer-exit-on-patch-failure": 1,
+    "patches": {
+      "squizlabs/php_codesniffer": {
+        "Fix conflict between scope indentation and array indentation checking. Ref. https://github.com/squizlabs/PHP_CodeSniffer/issues/2281": "https://gist.githubusercontent.com/drupol/c307b48fe4eb187b0833afb1d1b97e6a/raw/c3d414e9f4c182387ca9405c36f5daeb7b51ae05/phpcs-390bffe2265aac8184fdf86400c0d0841e65bc4b.patch",
+        "Disable exact checking of multi-line chained method calls in Generic.WhiteSpace.ScopeIndent. Ref. https://github.com/squizlabs/PHP_CodeSniffer/pull/2372": "https://gist.githubusercontent.com/drupol/c307b48fe4eb187b0833afb1d1b97e6a/raw/c3d414e9f4c182387ca9405c36f5daeb7b51ae05/phpcs-pr-2372.patch"
```

Comment:
Could we maybe add a section on the project's README.md where we explain the scope of this patches, when they can be removed (i.e. as soon that 3.5.0 will be released), and so on?

Reply:
Fixed.